src/main/resources/META-INF/injected-interfaces.json

```diff
@@ -270,5 +273,14 @@
   ],
   "net/minecraft/world/level/biome/BiomeSpecialEffects$GrassColorModifier": [
     "net/neoforged/fml/common/asm/enumextension/IExtensibleEnum"
+  ],
+  "net/minecraft/client/data/models/model/TexturedModel": [
```

Comment:
Please put all of these in the section above (preferably alphabetically). This section is for enum extension.

Reply:
Wasnt aware it was split into sections, moved my ifaces to above the first enum extension iface hopefully thats the correct spot.